src/wheatley.ts

```diff
@@ -642,17 +609,17 @@ export class Wheatley {
         return reply_message;
     }
 
-    is_root(user: Discord.User | Discord.PartialUser | Discord.APIUser): boolean {
-        //return member.roles.cache.some(r => r.id == root_role_id);
-        return root_ids.has(user.id);
+    is_root(user: Discord.User | Discord.PartialUser | Discord.GuildMember): boolean {
+        return this.roles.root.members.has(user.id) || user.id == this.user.id;
```

Comment:
I have a little FUD over relying on Role.members here since the docs refer to it as a cache and I don't know how that's populated. I'm going to look into it.